### PR TITLE
Allow uploading images from a url

### PR DESCRIPTION
**Pull request by rzane**
Carrierwave allows you to upload images from a url by simply surrounding the image parameter name with "remote" and "url". This PR just adds that input as well as a button to submit.

I don't speak Portugese, so I hope Google Translate was correct!

### CODE REVIEW

File: app/controllers/bootsy/images_controller.rb [hunk 1]
```diff
@@ -41,8 +41,8 @@ def destroy
 
     private
 
-    def find_gallery
-      ImageGallery.find(params[:image_gallery_id])
+    def set_gallery
+      @gallery = ImageGallery.find(params[:image_gallery_id])
```
Comment (volmer) [inline app/controllers/bootsy/images_controller.rb]: :heart:

File: app/views/bootsy/images/_new.html.erb [hunk 2]
```diff
@@ -4,5 +4,14 @@
...
   <%= image_tag 'bootsy/upload-loader.gif', class: 'bootsy-upload-loader' %>
 
+  <div class="input-group">
+    <%= f.url_field :remote_image_file_url, class: 'form-control', placeholder: t('bootsy.action.enter_image_url') %>
+    <span class="input-group-btn">
+      <%= f.submit t('bootsy.action.submit_upload'), class: 'btn btn-default' %>
+    </span>
+  </div>
+
+  <span>&nbsp;&nbsp;or&nbsp;&nbsp;</span>
```
Comment (volmer) [inline app/views/bootsy/images/_new.html.erb]: This padding should be set using css. We also need to i18n the text.

File: features/upload_image.feature [hunk 3]
```diff
@@ -11,6 +11,11 @@ Feature: Upload an image
     When I attach the file "test.jpg" on "image_file"
     Then I see the thumbnail "test.jpg" on the image gallery
 
+  Scenario: Upload a valid image from a URL
+    When I fill in "image[remote_image_file_url]" with "http://placehold.it/350x150"
```
Comment (volmer) [inline features/upload_image.feature]: Is this test performing an actual request?

**Comment by volmer** [comment]
Hey @rzane, thanks so much for the PR. It looks really good, I've just added a few comments. I also think it would be nice to have a validation somehow, in case someone enters an incorrect URL or something.

**Comment by rzane** [comment]
You're very welcome. Thanks for an easy-to-integrate wysiwyg editor!

The test was performing an actual http request. Forgot about that! I added the `sham_rack` gem to stub the HTTP interaction. I also added translations for 'or' and fixed the padding. I added validation in my other PR (#111).

**Comment by volmer** [comment]
Many, many thanks for this!

**Pull request merged**